server/controllers/questions.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable camelcase */
+import validator from '../helpers/validator';
+import createError from '../helpers/createError';
+import { questionQuery } from '../db/querydata';
+
+const success = (status, data) => ({ status, data });
+
+const convertVote = (vote) => {
+  const val = vote.toLowerCase();
+  if (val === 'upvote') return 1;
+  if (val === 'downvote') return -1;
+  return 0;
+};
+
+const control = {
+  getAll: async (req, res) => {
+    try {
+      const { meetupId } = await validator(req.params, 'reqId');
```

Comment:
@kossy360 What is `reqId`

Reply:
it's the key of the Joi schema object that validates all request parameters

Reply:
Okay.